src/StorageApiIndexClient.php

```diff
@@ -38,9 +50,4 @@ public function getServiceUrl(string $serviceName): string
     {
         return $this->getService($serviceName)['url'];
     }
-
-    protected function getTokenHeaderName(): ?string
```

Comment:
Takova subjektivni vec, ale kdyz tu byla ta metoda, prislo mi to takovy explicitnejsi, ze to nebude delat authentication.

Reply:
hm, nevím no, mě zas přišlo dementní mít mít metodu na getTokenHeader name a property na defaultHeaders a současně zbytečný dělat metodu na getDefaultHeaders

Reply:
jj, souhlas ze mit jedno tak a druhy tak je demence.. jen jsem se to pokusil pushnout smerem k tem metodam :wink: